build/aleth-interpreter.sh

```diff
@@ -2,6 +2,12 @@
 
 set -e
 
+
+if [[ "$OSTYPE" != "linux"* ]]; then
```

Comment:
```
echo $OSTYPE
linux-gnu

uname -a
Linux ubp52 5.2.9-050209-generic #201908160940 SMP Fri Aug 16 09:42:58 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux
```

Reply:
`[[ ! "$OSTYPE" =~ "linux" ]]`

Reply:
Ah, sorry @ziogaschr! My misread here. I didn't see the `*` in `"linux"*`. LGTM.